action/protocol/account/protocol.go

```diff
@@ -119,7 +119,9 @@ func createAccount(sm protocol.StateManager, encodedAddr string, init *big.Int)
 	case nil:
 		return errors.Errorf("failed to create account %s", encodedAddr)
 	case state.ErrStateNotExist:
-		account.Balance = init
+		if err := account.AddBalance(init); err != nil {
```

Comment:
can we add a `SetBalance()` to make it clear?

Reply:
SetBalance doesn't make sense. For new account, add balance is sufficient; for existing account, reset balance is not reasonable

Reply:
when will `ErrStateNotExist` be returned for a existing account?